doc/dask.rst

```diff
@@ -90,7 +92,7 @@ use :py:func:`~xarray.open_mfdataset`::
     xr.open_mfdataset('my/files/*.nc', parallel=True)
 
 This function will automatically concatenate and merge datasets into one in
-the simple cases that it understands (see :py:func:`~xarray.auto_combine`
+the simple cases that it understands (see :py:func:`~xarray.combine_by_coords`
```

Comment:
should that be `combine_nested` instead?

Reply:
The default is `combine="by_coords"` so this is right.